lib/src/input/figma/entities/layers/boolean_operation.dart

```diff
@@ -61,13 +62,13 @@ class BooleanOperation extends FigmaVector implements FigmaNodeFactory {
   @override
   Future<PBDLNode> interpretNode() {
     imageReference =
-        FigmaAssetProcessor().processImage(UUID, absoluteBoundingBox);
+        FigmaAssetProcessor().processImage(UUID, absoluteBoundingBox, name);
     return Future.value(PBDLImage(
       UUID: UUID,
       boundaryRectangle: absoluteBoundingBox?.interpretFrame(),
       isVisible: isVisible,
-      name: name,
-      style: style?.interpretStyle(),
+      name: AssetProcessingService.getImageName(name),
```

Comment:
Instead of calling the method `AssetProcessingService.getImageName(name)` every time we create a `PBDLImage`, couldn't we add this call in the `PBDLImage` constructor?

Something like 
```dart
PBDLImage(this.name){
    name = getImageName(name)
}
```

This way, we'll apply it to every instance of `PBDLImage`

Reply:
That is true